salt/modules/yumpkg.py

```diff
@@ -1769,7 +1771,8 @@ def upgrade(name=None,
     .. _`systemd-run(1)`: https://www.freedesktop.org/software/systemd/man/systemd-run.html
     .. _`systemd.kill(5)`: https://www.freedesktop.org/software/systemd/man/systemd.kill.html
 
-    Run a full system upgrade, a yum upgrade
+    .. versionchanged:: Fluorine
+        Added obsoletes and minimal arguments
```

Comment:
For emphasis, can you surround "obsoletes" and "minimal" with double-backticks? This will make them stand out in our documentation.

```rst
Added ``obsoletes`` and ``minimal`` arguments 
```